types/plotly.js/index.d.ts

```diff
@@ -856,6 +857,16 @@ export interface Margin {
     pad: number;
 }
 
+export interface ModeBar {
+    activecolor: Color;
+    add: ModeBarDefaultButtons[];
```

Comment:
docs say `ModeBarDefaultButtons | ModeBarDefaultButtons[]`:
```ts
Type: string or array of strings
```
```suggestion
    add: ModeBarDefaultButtons | ModeBarDefaultButtons[];
```